docker/templates/Dockerfile.erb

```diff
@@ -151,9 +151,8 @@ COPY pipeline/default.conf pipeline/logstash.conf
 RUN chown --recursive logstash:root config/ pipeline/
 # Ensure Logstash gets the correct locale by default.
 ENV LANG=<%= locale %> LC_ALL=<%= locale %>
-# Copy over the appropriate env2yaml artifact
-ARG TARGETARCH
-COPY env2yaml/env2yaml-${TARGETARCH} /usr/local/bin/env2yaml
+
+COPY env2yaml/env2yaml /usr/local/bin/env2yaml
```

Comment:
You probably have a phantom env2yaml file that was built a long time ago, so you are able to build the image.
Delete that file and rebuild everything should give `env2yaml-amd64` and `env2yaml-arm64` only